prelude/__internal__/System.mad

```diff
@@ -18,3 +18,14 @@ export exec = (command) => (#-
     })
   })
 -#)
+
+getArgs :: W.Wish String (List String)
+export getArgs = (#- W.of(process.argv.slice(2)) -#)
+
+getEnv :: String -> W.Wish String String
+export getEnv = (name) => (#- {
+  const envVar = process.env[name];
+  return envVar === undefined
```

Comment:
Minor (and likely bikeshedding) but I would prefer to have the imported code be closer to how we would write JS, i.e.
```js
return !envVar ? W.bad("Not found") : W.of(envVar)
```

Reply:
As I said on slack I think we want to differentiate between cases where we found an empty var and we we didn't find one.

Reply:
I am truly only saying `envVar === undefined` is the same as `!envVar` in a fenced context, because JS is coercive. But maybe we're talking about different things?